### PR TITLE
Add a prow job for updating oss prow monitoring stack

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

**Comment by chaodaiG** [comment]
/cc @mpherman2 @cjwagner 

This hasn't been tested yet, set it run weekly for now

File: prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml [hunk 1]
```diff
@@ -401,6 +401,37 @@ periodics:
...
+  extra_refs:
+  - org: GoogleCloudPlatform
+    repo: oss-test-infra
+    base_ref: master
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+  annotations:
+    testgrid-dashboards: googleoss-test-infra
+    testgrid-tab-name: auto-update-monitoring
```
Comment (mpherman2) [inline prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml:417]: nit: either name the tab autobump-monitoring or change the prowjob name to auto-update-monitoring to make it easier to understand which job the tab is for.

**Review by mpherman2 — APPROVED** [review]
/lgtm
/hold
just one small nit

File: prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml (hunk 1 above)
Comment (chaodaiG) [reply prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml]: done, thanks!

**Pull request merged**